### ISSUE TITLE
Will this remain open-source and maintained after the acquisition?

### ISSUE THREAD

**Issue by balupton** [darkskyapp/tz-lookup#61]
Sounds like Apple are putting a hammer to Dark Sky. I can have @bevry fork this library and take it over if it is going to disappear.

**Comment by ironwallaby** [comment]
Yes, this library is going to be put into read-only mode shortly. Please fork and let us know the URL, and we'll link to you in the README before winding down.

Thanks!
